tools/swarm/src/compose.rs

```diff
@@ -913,13 +893,13 @@ mod tests {
                 build: ./iroha-cloned
                 platform: linux/amd64
                 environment:
-                  IROHA_CHAIN_ID: 00000000-0000-0000-0000-000000000000
-                  IROHA_CONFIG: /config/config.json
-                  IROHA_PUBLIC_KEY: ed0120854457B2E3D6082181DA73DC01C1E6F93A72D0C45268DC8845755287E98A5DEE
-                  IROHA_PRIVATE_KEY: '{"digest_function":"ed25519","payload":"ab0e99c2b845b4ac7b3e88d25a860793c7eb600a25c66c75cba0bae91e955aa6854457b2e3d6082181da73dc01c1e6f93a72d0c45268dc8845755287e98a5dee"}'
-                  TORII_P2P_ADDR: 0.0.0.0:1340
-                  TORII_API_URL: 0.0.0.0:8083
-                  IROHA_GENESIS_PUBLIC_KEY: ed01203420F48A9EEB12513B8EB7DAF71979CE80A1013F5F341C10DCDA4F6AA19F97A9
+                  CHAIN_ID: 00000000-0000-0000-0000-000000000000
+                  PUBLIC_KEY: ed0120854457B2E3D6082181DA73DC01C1E6F93A72D0C45268DC8845755287E98A5DEE
+                  PRIVATE_KEY_DIGEST: ed25519
+                  PRIVATE_KEY_PAYLOAD: ab0e99c2b845b4ac7b3e88d25a860793c7eb600a25c66c75cba0bae91e955aa6854457b2e3d6082181da73dc01c1e6f93a72d0c45268dc8845755287e98a5dee
```

Comment:
I would prefer the old format. It should be possible to do in env variables

Reply:
Old format is this:

```shell
PRIVATE_KEY='{"digest_function":"ed25519","payload":"8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb"}'
```

I tried to avoid JSON in ENV, because it is messy, unreadable, and not _as_ easy to set as with direct values.

It would be also nice to get rid of necessity to specify trusted peers JSON list in ENV.